.github/CODEOWNERS

```diff
@@ -184,3 +184,10 @@
 /x-pack/test/api_integration/apis/endpoint/ @elastic/endpoint-app-team
 /x-pack/test/functional/apps/endpoint/ @elastic/endpoint-app-team
 /x-pack/test/functional/es_archives/endpoint/ @elastic/endpoint-app-team
+
+# SIEM
+/x-pack/legacy/plugins/siem/ @elastic/siem
+/x-pack/plugins/siem/ @elastic/siem
+/x-pack/test/detection_engine_api_integration @elastic/siem
+/x-pack/test/api_integration/apis/siem @elastic/siem
+/x-pack/plugins/case @elastic/siem
```

Comment:
nit: Looks like this file started out grouping alphabetically by owners -- optional to keep that pattern with SIEM

Reply:
It did but now it's so all over the place I would feel weird trying to order it by alphabetical without touching all the other areas. They stopped it alphabetically a while back. The "O" for example is stuck between two different spots. I recon this is what it is now and will just check in with our owners below the endpoint one.